frontend/src/APIClients/ReviewAPIClient.ts

```diff
@@ -1,3 +1,6 @@
+import { BEARER_TOKEN } from "../constants/AuthConstants";
+import baseAPIClient from "./BaseAPIClient";
+
 export type AuthorRequest = {
```

Comment:
What do you think about moving these types to `frontend/src/types/...`? I know some API clients do this while some keep their types coupled. We should probably decide on something consistent :thinking:

Reply:
Yeah I think it makes more sense to move these to `frontend/src/types/...` similar to auth and tags.